R/geom-rug.r

```diff
@@ -4,9 +4,10 @@
 #' with the two 1d marginal distributions. Rug plots display individual
 #' cases so are best used with smaller datasets.
 #'
-#' The rug lines are drawn with a fixed size (3\% of the total plot size) so
-#' are dependent on the overall scale expansion in order not to overplot
-#' existing data.
+#' By default, the rug lines are drawn with a length that corresponds to 3%
```

Comment:
You need to escape this `%` like the current one:

https://github.com/tidyverse/ggplot2/blob/9df0a076025aa14ec8cf079ae53e0d662884625a/R/geom-rug.r#L7

Reply:
Oops sorry! corrected